mds/provider_poller/settings.py

```diff
@@ -0,0 +1,5 @@
+from django.conf import settings
```

Comment:
I don't think we ever used that pattern of a settings module callling _the_ settings module.

That `getattr` could just go inside the code for a single setting. Or maybe just use appconf? (without the effing magic of the prefix!)

Reply:
I initially thought about appconf then I saw this pattern: https://github.com/Polyconseil/django-mds/blob/4fb79d15244ce76245ed75793affdee0773fdffd/mds/authent/settings.py
Have you played with it? Maybe it just does not work.

Reply:
My bad then, we actually use this pattern. But still probably overkill for a single setting, what about grouping with the ones from the keystore? https://github.com/Polyconseil/django-mds/blob/master/mds/provider_poller/oauth2_store.py#L17-L29

I put them in `mds/server/settings.py` but it just leads to duplicating them in plato. I didn't think about appconf.